src/commands/iam/tenant-iam-audit/get-audit-for-tenant.ts

```
@@ -0,0 +1,189 @@
+      // Handle empty responses
+      if (!rawResponse) {
+        return {
+          logs: [],
+          pagination: { page: 1, pageSize: 20, totalItems: 0, totalPages: 0 },
+        }
+      }
+
+      // Try the normal parsing
+      return parseResponseHelper(AuditLogResponseSchema, rawResponse)
+    } catch (error) {
+      throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)